### ISSUE TITLE
copy command doesn't recognise repo2 flag

### ISSUE THREAD

**Issue by djmattyg007** [creativeprojects/resticprofile#72]
`resticprofile` prompts for a `repo2` flag when running `copy` despite `repo2` already being specified in the config file.

**Comment by creativeprojects** [comment]
Well spotted 😄 
I should make a `copy` section of a profile, otherwise having a `repo2` parameter doesn't make sense to any other command

**Comment by creativeprojects** [comment]
If you're comfortable with compiling go code, the new feature is merged into `master`.

And here's how to configure it:

https://github.com/creativeprojects/resticprofile#special-case-for-the-copy-command-section

**Comment by djmattyg007** [comment]
I’m curious why it’s handled specially, as I thought anything that looked like a restic flag was passed straight through to restic.

Does this mean that if I want to copy to multiple different destinations, I need multiple profiles?

**Comment by djmattyg007** [comment]
Does the environment variable RESTIC_PASSWORD2 still work?

**Comment by creativeprojects** [comment]
> I’m curious why it’s handled specially, as I thought anything that looked like a restic flag was passed straight through to restic.

It's just for convenience, this config would also work:

```yaml
default:
    repository: "/backup/original"
    password-file: key
    copy:
        repo2: "/backup/copy"
        password-file2: other_key
```

> 
> Does this mean that if I want to copy to multiple different destinations, I need multiple profiles?

The way the configuration file is for now, yes. I do have plans to change the configuration file though, probably for a version 2.
(unless you can think of a way of doing it without breaking existing configurations...)

> Does the environment variable RESTIC_PASSWORD2 still work?

Yes it does. resticprofile would *add* environment variables, but would not interfere with the existing ones.

**Comment by djmattyg007** [comment]
To be clear, that configuration snippet in your most recent message only works with the patch from #73 right? I already tried that in the current version of resticprofile and it didn’t work, which is why I filed this issue.

**Comment by creativeprojects** [comment]
Released in v0.16.0
